pages/form/Question1.js

```diff
@@ -39,33 +38,55 @@ export default function Question1() {
         {children}
       </Checkbox>
     );
-  }
+  };
 
   const saveAnswers = (prev) => {
     setAnswers((prev) => ({
       ...prev,
       week: days,
-    }))
-  }
+    }));
+  };
 
   return (
     <Layout>
-      <Q1Progress /> 
+      <Q1Progress className="responsivebar" />
+      {/* <Q1Progress className="responsivebar" viewBox="100 5 490 50" height="50" width="100%" /> */}
```

Comment:
do we need this comment?
```suggestion
```

Reply:
This is the part you are mentioning need some work. I tried working on svg file. You know it an image pre fix with size. The image on the bar is what I'm struggling to get them responsive. If anyone can explain how I can do that?

Reply:
This is the part I'm trying to work on the svg bar and the image. I might need some help on the svg image on the bar if anyone familiar with making it responsive. The is a kind on seperate to normally how I have done with images to be responsive. I will still search up for tips.